packages/core-components/src/components/Link/Link.tsx

```diff
@@ -52,6 +53,20 @@ export type LinkProps = MaterialLinkProps &
     noTrack?: boolean;
   };
 
+export const useResolvedPath = (uri: LinkProps['to']) => {
+  const configApi = useApi(configApiRef);
```

Comment:
Ah, yeah.  Especially if we're already `use`'ing the config API!  We already have what we need.